src/modals/helpModal.js

```diff
@@ -4,6 +4,7 @@ const helpModal = (screen, blessed, application) => {
     ["-----------", "-----------"],
     ["Enter", "Displays function data when focused on Functions table"],
     ["o", "Opens the AWS console for a selected lambda function"],
+    ["o", "Opens the AWS console for a selected event bus"],
```

Comment:
Combine these two into one line Opens the AWS console for a selected lambda function or event bus